modules/virt-importing-vm-wizard.adoc

```diff
@@ -101,13 +101,21 @@ endif::[]
 ifdef::virt-importing-rhv-vm[]
 * *General* -> *Name*: The VM name is limited to 63 characters.
 * *General* -> *Description*: Optional description of the VM.
-* *Storage*:
 ** *Storage Class*: Select *NFS* or *ocs-storagecluster-ceph-rbd*.
 +
 If you select *ocs-storagecluster-ceph-rbd*, you must set the *Volume Mode* of the disk to *Block*.
++
+If you select *NFS*, you must run the following command on the CLI because of a known issue (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946177[*BZ#1946177*]):
```

Comment:
Not critical, but "on the CLI" sounds off to me. Would "from the CLI" work?

Reply:
I agree with you 100% but IBM Guide does not. See p. 216. :-(